### PR TITLE
fix(client): FAQ section now shows hand cursor

### PR DESCRIPTION
**Pull request by samahrens**
Close: #6302 

## PR Details

This change allows the user to hover over an FAQ question and see the hand pointer, notifying them that the question can be clicked and expanded.

Note: I wasn't sure how to use a clickable element (button, link) instead of changing the CSS, since buttons by default don't have the "cursor: pointer;" property.

### Summary
🖱️🖐️✏️

Improved the usability and appearance of the question component in the purchase page by making it fully clickable and adding a cursor style. Fixed a typo in a function name.

> _`question` component_
> _cursor and click area changed_
> _a winter bug fix_

### Walkthrough
*  Make the question component clickable and indicate the cursor style ([link](https://github.com/amplication/amplication/pull/6405/files?diff=unified&w=0#diff-9193771693484c4cc0a6dcdc9b65da0604028d5c18154d1c795f8cbc42166ddfR6), [link](https://github.com/amplication/amplication/pull/6405/files?diff=unified&w=0#diff-decb502b8f90a4e02431a131f6927b5dcec9c2cc13b0f9944de37d671a6365f9L17-R23))
  - Add `cursor: pointer` to the `.question` class in `Question.scss` ([link](https://github.com/amplication/amplication/pull/6405/files?diff=unified&w=0#diff-9193771693484c4cc0a6dcdc9b65da0604028d5c18154d1c795f8cbc42166ddfR6))
  - Move the `onClick` handler from the header div to the parent div of the question component in `Question.tsx` ([link](https://github.com/amplication/amplication/pull/6405/files?diff=unified&w=0#diff-decb502b8f90a4e02431a131f6927b5dcec9c2cc13b0f9944de37d671a6365f9L17-R23))
  - Fix the typo in the `handleOnClick` function name in `Question.tsx` ([link](https://github.com/amplication/amplication/pull/6405/files?diff=unified&w=0#diff-decb502b8f90a4e02431a131f6927b5dcec9c2cc13b0f9944de37d671a6365f9L17-R23))

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]

File: packages/amplication-client/src/Purchase/Question.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
 .question {
   border: 1px solid var(--black10);
   border-radius: 8px;
+  cursor: pointer;
```
[the remark this answers is not in the document]
Comment (samahrens) [reply packages/amplication-client/src/Purchase/Question.scss:6]: Hi, I can change it to a clickable element like a button/link, but buttons do not have a cursor pointer by default, so I would still have to add in CSS to fix it. Is there another way you are thinking of?

[the remark this answers is not in the document]
Comment (abrl91) [reply packages/amplication-client/src/Purchase/Question.scss:6]: @samahrens please, take a look at places where we use the `<Button>` component

[the remark this answers is not in the document]
Comment (samahrens) [reply packages/amplication-client/src/Purchase/Question.scss:6]: @mulygottlieb Please assign it to someone else if you can. Sorry, I've been busy the past few weeks!

[the remark this answers is not in the document]
Comment (mulygottlieb) [reply packages/amplication-client/src/Purchase/Question.scss:6]: No worries @samahrens . Thanks for the time you invested and the good will.

**Pull request closed without merging**